content/docs/use-cases/index.md

```diff
@@ -1,28 +1,28 @@
 # Use Cases
 
-We provide short articles on common ML workflow or data management scenarios
-that DVC can help with or improve. Our use cases are not written to be run
-end-to-end like tutorials. For more general, hands-on experience with DVC,
-please see our [Get Started](/doc/tutorials/get-started) instead.
+We provide short articles on common ML workflow and data science use cases that
+DVC can help with or improve. Our use cases are not written to be run end-to-end
+like tutorials. For more general, hands-on experience with DVC, please see our
+[Get Started](/doc/tutorials/get-started) instead.
 
 ## Why DVC?
 
 Even with all the success we've seen today in machine learning (ML), especially
-with deep learning and its applications in business, the data science community
-still lacks good practices for organizing their projects and collaborating
-effectively. This is a critical challenge: while ML algorithms and methods are
-no longer tribal knowledge, they are still difficult to implement, reuse, and
-manage.
+with deep learning and its applications in business, data scientists still lack
+good tools for organizing their projects and collaborating effectively. This is
+a critical challenge: while ML algorithms and methods are no longer tribal
+knowledge, they are still difficult to implement, reuse, and manage.
 
 ## Basic uses of DVC
 
 If you store and process data files or datasets to produce other data or machine
 learning models, and you want to
 
 - capture and save <abbr>data artifacts</abbr> the same way you capture code;
-- track and switch between different versions of data or models easily;
-- understand how data or models were built in the first place;
-- be able to compare models and metrics to each other;
+- track, control, and switch between different versions of data or models
+  easily;
+- understand how machine learning models were built in the first place;
+- compare ML models and metrics to each other;
```

Comment:
- OK with introducing "control" if we need it near "version", but if you want you can also add "version control" somewhere else in the doc or even in a new sentence if needed (not sure where).
- "machine learning models" 👍 (why not ML though? Up to you) but why was "data or" removed? The data part is also important, probably more that the models part 🙂

Reply:
> * OK with introducing "control" if we need it near "version", but if you want you could just add "version control" somewhere else in the doc instead, or even in a new sentence if needed (not sure where).
OK to have within proximity. 
> * "machine learning models" 👍 (why not ML though? Up to you) but why was "data or" removed? The data part is also important, probably more that the models part 🙂
The next bullet has ML, and we should have both. I thought the "data or" might be a typo because "...how data or models were built in the first place" implies "how data was built". Is that what we're trying to say or should it be "datasets or models"?

Reply:
It's not a typo. We have "data and (ML) models" repeatedly throughout docs. Not sure what's wrong about "data was built". "Datasets" may be more correct in some ways but we want to have the term "data" a lot. Anyway, not something within the scope of this PR to decide on, let's just revert that part please.

Reply:
OK, understood. Updated to revert and adjust ML/machine learning placements. PTAL

Reply:
Thanks.

Reply:
BTW don't get me wrong, no problem in suggesting any changes you feel are necessary (just try to respect the scope more or less). I/we can always just ask to revert but that's normal 🙂